employees/tests/test_unit_report_viewset.py

```diff
@@ -0,0 +1,143 @@
+from django.test import TestCase
+from rest_framework.reverse import reverse
+from employees.models import Report
+from users.models import CustomUser
+from managers.models import Project
+import datetime
+from decimal import Decimal
+from employees.views import ReportViewSet
+from rest_framework.test import APIRequestFactory
+from django.contrib.auth.models import AnonymousUser
```

Comment:
Sort these imports in more sensible way. employees adjacent to employees. datetime, decimal at the top, alphabetical order and so one.

Reply:
Should I put blank line between each group like PEP8 suggests? If so, what is defined as a group? Standard-Third party-Local? Or maybe per module?

```
import datetime
from decimal import Decimal

from django.contrib.auth.models import AnonymousUser
from django.test import TestCase
from rest_framework.reverse import reverse
from rest_framework.test import APIRequestFactory

from employees.models import Report
from employees.views import ReportViewSet
from managers.models import Project
from users.models import CustomUser
```

Like that?

Reply:
This is even prettier ;)

Reply:
Done